mlir/tools/mlir-pet/islNodeBuilder.cc

```diff
@@ -29,19 +29,72 @@ static int createIntFromIslExpr(isl::ast_expr expression) {
 // TODO: See how we can get location information.
 // TODO: handle degenerate loop (see isl_ast_node_for_is_degenerate)
 // TODO: See how to handle more complex expression in the loop.
+
+bool isInt(isl::ast_expr expression) {
```

Comment:
Please remove "if". Just write return isl_ast_expr_get_type(expression.get()) == isl_ast_expr_in in the body of the function.